rubocop-shopify.gemspec

```
@@ -20,5 +20,5 @@ Gem::Specification.new do |s|
     "source_code_uri"   => "https://github.com/Shopify/ruby-style-guide/tree/v#{s.version}",
   }
 
-  s.add_dependency "rubocop", ">= 0.78.0"
+  s.add_dependency "rubocop", "~> 0.81.0"
```

```suggestion
  s.add_dependency "rubocop", ">= 0.81.0"
```

If we don't allow people to install versions of rubocop that would break this gem we would never get PR like this anymore. Dependabot will not upgrade rubocop automatically in apps, and apps would not know that by upgrading rubocop  their CI fail.

This is without upper limit by design, no mistake. The upper limit doesn't protect anymore and break the automatic upgrade of gems.